index.js

```diff
@@ -15,6 +15,11 @@ var PluginError = gutil.PluginError;
 var PLUGIN_NAME = 'gulp-istanbul';
 var COVERAGE_VARIABLE = '$$cov_' + new Date().getTime() + '$$';
 
+function getNormalizedPath(filepath) {
+  if (path.sep !== '\\') return filepath;
+  return filepath.replace(/\//g, '\\');
```

Comment:
Clarity is better than performance.

You shouldn't care about performance until it become an issue.